ci-operator/step-registry/ipi/conf/aws/custom-az/ipi-conf-aws-custom-az-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+export AWS_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/.awscred"
+REGION="${LEASED_RESOURCE}"
+CONFIG="${SHARED_DIR}/install-config.yaml"
+
+function join_by { local IFS="$1"; shift; echo "$*"; }
+
+mapfile -t AVAILABILITY_ZONES < <(aws --region "${REGION}" ec2 describe-availability-zones | jq -r '.AvailabilityZones[] | select(.State == "available") | .ZoneName' | sort -u)
+ZONES=("${AVAILABILITY_ZONES[@]:0:${ZONES_COUNT}}")
+ZONES_STR="[ $(join_by , "${ZONES[@]}") ]"
+echo "AWS region: ${REGION} (zones: ${ZONES_STR})"
+
+CONFIG_PATCH="${SHARED_DIR}/install-config-zones.yaml.patch"
+cat > "${CONFIG_PATCH}" << EOF
+controlPlane:
+  platform:
+    aws:
+      zones: ${ZONES_STR}
+compute:
+- platform:
+    aws:
+      zones: ${ZONES_STR}
+EOF
+  yq-go m -x -i "${CONFIG}" "${CONFIG_PATCH}"
```

Comment:
some redundant leading white spaces on this line?

Reply:
fixed